week-1/implement/sum.js

```diff
@@ -0,0 +1,17 @@
+function sumNumber(array) {
+    if(array.length===0){
+        return 0;
+    }
+    if(array.length===1){
```

Comment:
You don't really need these two if statements because the for statement below will work if the length of the array is equal to 0 and when it is equal to 1.  Your code works correctly, but less code is better, there isn't really a need here to have a special case for either length 0 or length 1.
Lines 8 to 13 are all you need in this function.